static/styles/rendered_markdown.css

```diff
@@ -309,8 +309,22 @@
         margin-bottom: 5px;
         margin-left: 5px;
         height: 100px;
-        display: block !important;
-        border: none !important;
+
+        /* We need an explicit width on the containers to make sure our layout doesn't jump around as images are loaded! */
+        width: 150px;
+
+        /* The containers also need an inline-block display so they don't take up an entire row on their own; only what they need. */
+        display: inline-block !important;
```

Comment:
Why does this need `!important`?  I'd expect this CSS to take precedence over other `display` values.

Reply:
I removed the `!important` in my push; the other changes are all comments/reordering.